cantabular-import/dp-cantabular-xlsx-exporter.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.3'
+services:
+    dp-cantabular-xlsx-exporter:
+        build:
+            context: ../../dp-cantabular-xlsx-exporter
+            dockerfile: Dockerfile.local
+        command:
+            - reflex
+            - -d
+            - none
+            - -c
+            - ./reflex
+        volumes:
+            - ../../dp-cantabular-xlsx-exporter:/dp-cantabular-xlsx-exporter
+        depends_on:
+            - kafka
+            - minio
+        ports:
+            - 26800:26800
+        restart: unless-stopped
+        environment:
+            BIND_ADDR:                  ":26800"
+            KAFKA_ADDR:                 "kafka:9092"
+            DATASET_API_URL:            "http://dp-dataset-api:22000"
+            LOCAL_OBJECT_STORE:         "http://minio:9000"
+            MINIO_ACCESS_KEY:           "minio-access-key"
+            MINIO_SECRET_KEY:           "minio-secret-key"
+            SERVICE_AUTH_TOKEN:         $SERVICE_AUTH_TOKEN
+            VAULT_TOKEN:                "0000-0000-0000-0000"
+            UPLOAD_BUCKET_NAME:         "dp-cantabular-csv-public"
```

Comment:
For CSV exporter, David is now using what i now also have in the XLSX exporter's config:
		UploadBucketName:           "public-bucket", // where to place the created .xlsx
		PrivateUploadBucketName:    "private-bucket",

And i don't have UPLOAD_BUCKET_NAME & PRIVATE_UPLOAD_BUCKET_NAME in my version of this file as it messed things up.
So, please remove lines 30 and 31 for XLSX exporter to correctly interact with the CSV exporter.

Reply:
Ok. The buckets haven't been updated on the minio config so I will need to do that too. I was going to suggest we switch to just 2 buckets across the export services, glad you had the same idea.

Reply:
I'm just following along with David's idea/work.